.travis.yml

```diff
@@ -1,11 +1,12 @@
+---
 language: python
 dist: bionic
 python:
   - "2.7"
```

Comment:
We can remove python2.7 compatibility. Make sure to update the `python_requires` marker accordingly in the setup.py.

Reply:
We can make it in #105 (it just needs some small polishing) after merging this one. What do you think?